src/geo/gmaps/gmaps-default-map-style.js

```diff
@@ -1,69 +0,0 @@
-var DEFAULT_MAP_STYLE = [{
```

Comment:
That's something we'll have to think we work on the API. Anyway, if someone wanted to create a map from a custom viz.json for example, they [should be able to specify the `baseType` and `style` attributes](https://github.com/CartoDB/cartodb.js/blob/ff9d12ad6f78035d42dd224cc37c268346e7a159/src/geo/gmaps/gmaps-base-layer-view.js#L31-L40) to customise their maps. And, [they will get the default style if they use any of the default map types](https://github.com/CartoDB/cartodb.js/blob/ff9d12ad6f78035d42dd224cc37c268346e7a159/src/geo/gmaps/gmaps-base-layer-view.js#L6-L11).